src/local-editor-service.ts

```diff
@@ -295,6 +327,30 @@ export class LocalEditorService extends EditorService {
 
       return concatMap(elements, (el) => el.attributes)
           .find(at => at.name === location.attribute);
+    } else if (location.kind === 'attributeValue' || location.kind === 'text') {
+      const domModules = document.getByKind('dom-module');
+      for (const domModule of domModules) {
+        const elements =
```

Comment:
`document` is an indexed analyzer output, and we're querying the high level objects that it has found. Here's the interface: https://github.com/Polymer/polymer-analyzer/blob/master/src/model/queryable.ts#L42